src/query-filter.ts

```diff
@@ -0,0 +1,32 @@
+export interface Query {
+  /**
+   * sortType
+   */
+  sortType:
+    | "recency"
+    | "title"
+    | "author"
+    | "acquisition_desc"
+    | "acquisition_asc";
+}
+
+export interface Filter {
+  /**
+   * Filter by abonnement.
+   */
+  originType?: "KINDLE_UNLIMITED" | "PRIME" | "COMICS_UNLIMITED";
+  /**
+   * Request the next page of results.
+   */
+  paginationToken?: string;
+  /**
+   * Set the number of results to return.
+   * Default is 50.
+   */
+  querySize?: number;
+  /**
+   * When false, pagination is enabled and all results are returned.
+   * The default is false and only the first page of results is returned.
+   */
+  firstPageOnly?: boolean;
```

Comment:
I think this should read `The default is true`?

Reply:
Ah, sorry. I've changed the field name and description several times. Nothing felt quite right...
Nevertheless I've updated the PR with a refined version.